build/csslint/xfail.list

```diff
@@ -104,6 +104,7 @@ apps/wappush/style/wappush.css 0 1
 shared/elements/gaia_checkbox/style.css 5 0
 shared/elements/gaia_confirm/style.css 4 0
 shared/elements/gaia_grid/style.css 0 5
+shared/elements/gaia_menu/style.css 3 0
```

Comment:
Yeah, I don't like that either... Everything should go back to normal when [bug 1207131](https://bugzilla.mozilla.org/show_bug.cgi?id=1207131) will land.